src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -489,37 +486,6 @@ void remove(int index) {
 		modified();
 	}
 
-	@Override
-	public Iterator<ItemStack> containerIterator() {
```

Comment:
do you think any addons used this? maybe this should be deprecated for removal

Reply:
1) I didn't know it existed until today. (SkBee is the biggest add-on and doesn't touch it, I cant imagine another add-on touching it)
2) This bug actually caused an issue in SkBee (ExprAvailableMaterials - wasn't even looping ItemType but still broke it)
3) I could deprecate it, but it doesn't work. I guess I could have it return null (for the itemstack) instead of throwing an error.

Reply:
aight fair enough

Reply:
I ended up bring it back anyways, and just forwarded it to #getAll .... just in case.